.eslintrc

```diff
@@ -1,6 +1,7 @@
 {
 	"env": {
-		"node": true
+		"node": true,
+    "es6": true
 	},
 	"plugins": [
 		"nodeca"
```

Comment:
Yup please, if you end up with a lot of errors you can correct most of them with `$(npm bin)/eslint --fix`